inductiva/structures/post_processing/fenicsx.py

```diff
@@ -0,0 +1,149 @@
+"""Post-process FEniCSx simulation outputs."""
+
+import os
+import tempfile
+
+import pyvista as pv
+
+import inductiva
+
+
+class FEniCSxSimulationOutput:
```

Comment:
The name of this class is too generic for FEniCSx, but actually your rendering is specific for the structures case. I think you won't be able to render all FEniCSx simulation scenarios with it, right?

Reply:
If it's correct, it depends on what you're saving, whether it's displacements, stresses, strains, I'll renaming the class, thanks Ivan!

Reply:
But If I do another scenario and I provide the displacements, stresses and strains. Will I be able to use this?